battleship.py

```diff
@@ -18,55 +24,108 @@ def clear_grid():
         new_grid.append([])
     for i, k in enumerate(new_grid):
         for a in range(grid_size):
-            new_grid[i].append("0")
+            new_grid[i].append("-")
     return new_grid
 
 
+def check_ships(ask, ships):
+    if ask == [-1, -1]:
+        return True
+    elif ask in ships:
+        return False
+    else:
+        return True
+
+
 def place_ships():
```

Comment:
We need to discuss a new algorithm for this function as soon as we have most of the game working.